src/migrations/versions/dbb81bd4d47f_add_jobagent.py

```diff
@@ -5,34 +5,41 @@
 """
 from alembic import op
 import sqlalchemy as sa
+from sqlalchemy import inspect
 
 
+from config import app_config
+
 # revision identifiers, used by Alembic.
 revision = "dbb81bd4d47f"
 down_revision = "cbbba858deb0"
 branch_labels = None
 depends_on = None
 
+DATABASE_URL = app_config.database.url
+engine = sa.create_engine(DATABASE_URL)
+
 
 def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table(
-        "jobagent",
```

Comment:
similarly, please don't change and reformat this file

Reply:
Ok